ownCloud/Client/ClientRootViewController.swift

```diff
@@ -139,10 +144,16 @@ class ClientRootViewController: UITabBarController {
 	override func viewDidLoad() {
 		super.viewDidLoad()
 
-		filesNavigationController = ThemeNavigationController()
-		filesNavigationController?.navigationBar.isTranslucent = false
-		filesNavigationController?.tabBarItem.title = "Browse".localized
-		filesNavigationController?.tabBarItem.image = Theme.shared.image(for: "folder", size: CGSize(width: 25, height: 25))
+        self.view.backgroundColor = Theme.shared.activeCollection.tableBackgroundColor
+        
+        self.navigationController?.setNavigationBarHidden(true, animated: true)
```

Comment:
@hosy do you intend `animated: true` here in `viewDidLoad`?

Reply:
Changed my indent behavior in Xcode from Spaces to Tabs no.